src/map/map.ts

```diff
@@ -535,8 +535,8 @@ export class PropertiesMap {
                 color: this._colors(0)[0],
                 coloraxis: 'coloraxis',
                 line: {
-                    color: this._lineColors(0)[0],
-                    width: 1,
+                    color: 'black',
+                    width: this._options.markerOutline.value ? 0.5 : 0,
```

Comment:
I'm not sure this really solve the issue in https://github.com/lab-cosmo/chemiscope/issues/369#issuecomment-2400722768: if there are a lot of black points together, they will still smudge together.

I'm fine with reducing the width to 0.5, but I would rather use black outline for light colors and a grey outline (since the background is white) for dark colors. I'm sure there is a package to compute color luminance in JS.

Reply:
I agree with you that it should not be a fully customizable plotting code, but you are the one adding a customization option here 😃 Anyway, seems that most people want the option, so let's do it.